tests/components/devialet/test_init.py

```diff
@@ -14,20 +16,21 @@ async def test_load_unload_config_entry(
     hass: HomeAssistant, aioclient_mock: AiohttpClientMocker
 ) -> None:
     """Test the Devialet configuration entry loading and unloading."""
-    entry = await setup_integration(hass, aioclient_mock)
+    with patch("devialet.DevialetApi.upnp_available", return_value=True):
```

Comment:
Please patch the library to where you use it

Reply:
Without this patch it will use socket to discover UPnP (only needed when the mock is setup with `state="playing"`).